### PR TITLE
hiii

### LINKED ISSUE #1: Docker Exercise
Do it now

### PR DESCRIPTION
**Pull request by vijayrr22**
Fixes #1

### CODE REVIEW

**Review by sai-siddartha-d — CHANGES_REQUESTED** [review]
Give valid commit messages

**Comment by vijayrr22** [comment]
okay thanks for aprroving

File: .github/workflows/docker.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Docs for the Azure Web Apps Deploy action: https://github.com/Azure/webapps-deploy
+# More GitHub Actions for Azure: https://github.com/Azure/actions
+
+name: Build and deploy container app to Azure Web App - dpc-container-app
```
Comment (vijayrr22) [inline .github/workflows/docker.yml:4]: hiii

**Review by sai-siddartha-d — APPROVED** [review]
Looks Good

**Pull request merged**